Cargo.toml

```diff
@@ -23,6 +23,7 @@ rand = "0.5"
 rustyline = { version = "1", optional = true }
 y4m = { version = "0.3", optional = true }
 backtrace = "0.3"
+syn = "^0.15.20"
```

Comment:
'^' ?

Reply:
it allows all versions above 0.15.20 that are still 0.15

Reply:
oh...

Reply:
"Version at least"